src/utils/utils.jl

```diff
@@ -563,3 +563,34 @@ transform_array_for_hdf(
 
 transform_array_for_hdf(data::Vector{T}) where {T <: FunctionData} =
     throw(ArgumentError("Not currently implemented for $T"))
+
+to_namedtuple(val) = (; (x => getfield(val, x) for x in fieldnames(typeof(val)))...)
+
+function compute_file_hash(path::String, files::Vector{String})
```

Comment:
If there is a tracking list for reorganization, this file needs to be split up into different files with appropriate names. It’s currently a dumping ground.

Reply:
We should open an issue for tracking missing items.